test/integration/auth.spec.ts

```diff
@@ -1302,6 +1325,12 @@ describe('admin.auth', () => {
       multiFactorConfig: {
         state: 'ENABLED',
         factorIds: ['phone'],
+        providerConfigs: [
```

Comment:
I think we need one more struct -expectedUpdatedTenantSmsEnabledTotpDisabled to cover the case where TOTP is disabled via provider config, but it preserves the phone MFA settings.